Automattic/sensei
# Rename event logging JS file to match file naming standards

Changes the JS filename to use a hyphen instead of an underscore.

## Testing Instructions

Ensure the JS events are still being logged after this change.

- https://github.com/Automattic/sensei/pull/2715
- https://github.com/Automattic/sensei/pull/2702
- https://github.com/Automattic/sensei/pull/2697
- https://github.com/Automattic/sensei/pull/2675
